core-bundle/src/Routing/RouteProvider.php

```diff
@@ -520,7 +526,7 @@ private function findRootPages(string $httpHost): array
             $rootPages = $pages->getModels();
         }
 
-        $pages = $pageModel->findBy(["tl_page.alias='index' OR tl_page.alias='/'"], null);
+        $pages = $pageModel->findBy(["tl_page.alias IN ('index', '/')"], null);
```

Comment:
What is the reason for this change?

Reply:
Ah, seems we have already discussed this in https://github.com/contao/contao/pull/1776/files#r432486188. 😄